00-getting-started.md

```diff
@@ -119,7 +119,7 @@ You can also test running example. Save below code as `index.html` and open it w
 
 ### For bootstrap 4
 
-You can also use Summernote with Bootstrap 4 using `summernote-bs4.js` and `summernote-bs4.css`. This is also beta version, as Bootstrap 4 is in beta. Below is a code example using bootstrap 4.
+You can also use Summernote with Bootstrap 4 using `summernote-bs4.js` and `summernote-bs4.css`. Note that this is a beta version. Below is a code example using Bootstrap 4.
```

Comment:
The version of Summernote being used on the Website isn't in Beta, it's the 0.8.12 release. At  guess whoever wrote the text you've changed considered that the Summernote version using BS4 should be considered being in Beta as it was likely that the BS4 features could change at the time. So we should get rid of the Beta all together.